cerberus/kubernetes/client.py

```diff
@@ -80,6 +82,41 @@ def check_sdn_namespace():
         please specify the correct networking namespace in config file")
 
 
+def namespace_sleep_track(namespace):
+    global pods_tracker
+    pods = list_pods(namespace)
+    crashed_restarted_pods = []
+    for pod in pods:
+        try:
+            pod_info = cli.read_namespaced_pod_status(pod, namespace,
```

Comment:
Make this a separate function and call it to provide the given pod status? This way we can reuse it in other places as well. Thoughts?